.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: test
+
+on:
```

Comment:
This will trigger the action only on main, I find that developers often prefer having the CI run on their feature branches as well, so branches: '*' is better

Reply:
One question here Aki.

I made the changes here:
```
  push:
    branches: [ '*' ]
  pull_request:
    branches: [ main ]
```
And when I pushed the committed it to trigger two runs:

<img width="932" alt="Screen Shot 2022-05-06 at 16 28 05" src="https://user-images.githubusercontent.com/79212398/167204857-7ccd3dce-d97b-4fc0-bac0-dd3cd7bf0f13.png">

Maybe it is better to do:

```
  push:
    branches: [ main ]
  pull_request:
    branches: [ '*' ]
```

What do you think?